### ISSUE TITLE
[Test failed] NameError: name '_B' is not defined

### ISSUE THREAD

**Issue by kevinsmia1939** [hyperspy/link_traits#2]
Hello,

I am trying to package link_traits 1.0.2 into openSUSE Tumbleweed repository.
Building the package on headless server on openSUSE Tumbleweed failed during testing.
Using pytest to test the package. 
Any ideas?

```
[   46s] ==================================== ERRORS ====================================
[   46s] _______________ ERROR collecting link_traits/test_link_traits.py _______________
[   46s] link_traits/test_link_traits.py:24: in <module>
[   46s]     class TestLinkBidirectional:
[   46s] link_traits/test_link_traits.py:45: in TestLinkBidirectional
[   46s]     def test_connect_same_trait_source_undefined(self, A=_A, B=_B):
[   46s] E   NameError: name '_B' is not defined
[   46s] !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
[   46s] =============================== 1 error in 0.59s ===============================
```
Full log here: https://build.opensuse.org/package/live_build_log/home:andythe_great/python-link-traits/openSUSE_Tumbleweed/x86_64

Thanks.

**Comment by francisco-dlp** [comment]
Thanks for reporting this issue. I've just fixed it in the latest release, v1.0.3, that is already in pypi. Could you confirm that it now works for you?

**Comment by kevinsmia1939** [comment]
> Thanks for reporting this issue. I've just fixed it in the latest release, v1.0.3, that is already in pypi. Could you confirm that it now works for you?

The issue is fix now, didn't know I also need traitlets as a dependency, do I need it in the latest link_traits version?

**Comment by francisco-dlp** [comment]
`traitlets` is not required unless you plan to link traits to traitlets, that's why it's not in requirements. The fact that the tests failed without traitlets was a but, thanks for reporting it.
